content/lang/tr/about/demos-examples.md

```diff
@@ -60,40 +60,40 @@ TodoMVC için çevrimdışı senkronizasyon PuchDB ile birlikte [PouchDB](https:
 [Github Projesi](https://github.com/katopz/preact-todomvc-pouchdb)
 
 **[Hacker News Minimal](https://developit.github.io/hn_minimal/)** :newspaper:  
-Minimal olarak hazırlanmış hacker news web sitesi.  
+Oldukça miminal olarak hazırlanmış hacker news web sitesi.  
 [Github Projesi](https://github.com/developit/hn_minimal)
 
 **[Preact Boilerplate](https://preact-boilerplate.surge.sh)** :zap:  
-2 command starter project. Preact + Webpack + LESS + CSS Modules.  
-[Github Project](https://github.com/developit/preact-boilerplate)
+Preact + Webpack + LESS + CSS paketlerinin dahil olduğu başlangıç paketi. 
+[Github Projesi](https://github.com/developit/preact-boilerplate)
 
-**[Preact Offline Starter](https://preact-starter.now.sh)** :100:  
-Simplified Webpack2 starter for Progressive Web Apps, with offline support.  
-[Github Project](https://github.com/lukeed/preact-starter)
+**[Preact Çevrimdışı Başlangıç Kiti](https://preact-starter.now.sh)** :100:  
+Webpack2 kullanan Progressive Web Apps ve çevrımdışı destekli başlangıç paketi.  
+[Github Projesi](https://github.com/lukeed/preact-starter)
 
-**[Preact Redux Example](https://preact-redux-example.surge.sh)** :repeat:  
-Preact + Redux example project, implementing a simple To-Do list.  
-[Github Project](https://github.com/developit/preact-redux-example)
+**[Preact Redux Örneği](https://preact-redux-example.surge.sh)** :repeat:  
+Preact + Redux örnek projesi, basit bir to-do list uygulaması.  
```

Comment:
@irfansener burda uygulaması değil uygunlanması demek istemiş. 
"Preact + Redux örnek projesi, basit bir to-do list'e uygulanması."
daha doğru bir kullanım